.travis.yml

```diff
@@ -11,14 +11,16 @@ dist: bionic
 services:
   - docker
 
+env:
+  global:
+    - FAF_DB_VERSION="v93"
```

Comment:
Is it tag name? If yes, i would name variable with suffix _tag

Reply:
I guess. I mean technically it can be anything that can follow `git checkout` so it could be a commit hash or something. Although in practice we will probably always put a tag there

Reply:
I'd name it with tag suffix, cause you mean it to be tag.